packages/vite/src/node/plugins/resolve.ts

```diff
@@ -302,10 +314,12 @@ export function resolvePlugin(resolveOptions: InternalResolveOptions): Plugin {
 
       // external
       if (isExternalUrl(id)) {
-        return {
-          id,
-          external: true,
-        }
+        return options.idOnly
+          ? id
+          : {
+              id,
+              external: true,
+            }
```

Comment:
micro nit: by it looks nicer if this is a single line 😄

Reply:
ha, good one, done!